packages/react-components/src/components/ChatMessage/ChatMessageComponent.tsx

```diff
@@ -52,6 +52,11 @@ type ChatMessageComponentProps = {
    * @param userId - user Id
    */
   onRenderAvatar?: OnRenderAvatarCallback;
```

Comment:
I am a little bit curious about what layer customers are asking for the dateTimeFormat? Is it component layer or composite layer?

Reply:
It should really be in both. The components need to handle it and we want to expose it at the composite level.

@PorterNan  can we confirm this will be exposed through the JS bundle as well?